client/my-sites/comments/controller.js

```diff
@@ -118,9 +118,11 @@ export const comment = ( { query, params, path, store } ) => {
 	}
 
 	const action = sanitizeQueryAction( query.action );
+	const redirectToPostView = postId => () =>
+		page.redirect( `/comments/all/${ siteFragment }/${ postId }` );
 
 	renderWithReduxStore(
-		<CommentView { ...{ action, commentId, siteFragment } } />,
+		<CommentView { ...{ action, commentId, siteFragment, redirectToPostView } } />,
```

Comment:
I'd stick with one param name, but with a named value, eg:
```
deleteRedirect: redirectToPostView
```
So at a glance we know someone reading can guess that we redirect to post view, after a delete.